test/integration/users_login_test.rb

```diff
@@ -25,8 +25,11 @@ def setup
 
   test "login with valid information followed by logout" do
     get login_path
-    post login_path, params: { session: { email: @user.email,
-                                          password: 'password' } }
+    post login_path, params: { session: 
+                              { email: @user.email,
+                                password: 'password' 
+                                } 
+                              }
```

Comment:
as suggested in previous PR, rather assign the params hash to a variable (with 2-space indentation) and use that here `, params: params`